tools/go-cli/go-whisk-cli/commands/package.go

```diff
@@ -185,16 +184,6 @@ var packageCreateCmd = &cobra.Command{
         }
         client.Namespace = qName.namespace
 
-        if (flags.common.shared == "yes") {
```

Comment:
@rabbah - Do you think going forward this shared argument will eventually take additional states besides today's binary public|private values?  IMHO, I think we'll see additional shared scopes, including custom ones, in the future.  If so, then perhaps we should move this argument away from being a true|false state and more of a select-a-shared-scoped kind of argument.